tests/helpers/drupal-commands.js

```diff
@@ -94,7 +94,7 @@ module.exports = {
    */
   checkWatchdogErrors: async (timestamp, fail_on_notice) => {
     const result = drush(`test:checkWatchdog "${timestamp}" "${fail_on_notice}"`);
-    return result.toString();
+    return result;
```

Comment:
I've made it work with this:

`return parseInt(result.toString().replace(/\D/g, ''));`